src/app/modules/payments/stripe.controller.ts

```diff
@@ -191,10 +241,7 @@ export const checkPaymentReceiptStatus: ControllerHandler<{
     },
   })
 
-  return StripeService.getPaymentFromLatestSuccessfulCharge(
```

Comment:
This change has conflict with [dedicated payment page](https://github.com/opengovsg/FormSG/pull/6001/files#diff-a8cc140505581213ebf38ff6c87b84cba6e125176f67467358811b15a684e747L180).

Reply:
oh no HAHA ok, will update to be consistent with yours

Reply:
Yeah, I think its better for yours to merge first (as option 1), and then we re-evaluate it on the other PR.